subsys/bluetooth/services/fast_pair/fp_storage/fp_storage_manager.c

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <errno.h>
+#include <string.h>
+#include <zephyr/settings/settings.h>
+#include <bluetooth/services/fast_pair.h>
+
+#include "fp_storage_manager.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(fp_storage_manager, CONFIG_FP_STORAGE_LOG_LEVEL);
+
+#define SETTINGS_SM_SUBTREE_NAME "fp_sm"
+#define SETTINGS_RESET_IN_PROGRESS_NAME "reset_in_progress"
```

Comment:
Maybe we can call it `reset` (to make it shorter and save some space in settings)

Reply:
Ok, but in other places (variables/defines names) I would prefer to keep the `_in_progress` suffix.